src/Features/Core/Portable/ExternalAccess/Pythia/Api/PythiaSymbolMatchPriority.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.Completion.Providers;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Pythia.Api
+{
+    internal static class PythiaSymbolMatchPriority
+    {
+        internal static int Keyword = SymbolMatchPriority.Keyword;
```

Comment:
💡 Make these read-only and/or constant

Reply:
Making it read-only. Constant would be embedded to the target assembly, not sure if we want that.